app/ante/utils_test.go

```diff
@@ -91,40 +89,6 @@ func (suite *AnteTestSuite) CreateTestTx(
 	builder.SetFeeAmount(fees)
 	builder.SetGasLimit(msg.GetGas())
 
-	// First round: we gather all the signer infos. We use the "set empty
```

Comment:
we should provide a bool to the function if we want to sign with the Cosmos format and adapt the tests to work with both cases

Reply:
fixed now, the old tests are still run against signed tx, added new test case for un-signed tx.